### PR TITLE
INFRA-2480 [not-for-merge] [#just-testing] Test updated jdk8 adopt-openjdk

### PR DESCRIPTION
**Pull request by timja**
Testing adoptopenjdk latest java 8 on ci.jenkins.io, as the current jdk is oracle based and quite old...

https://issues.jenkins-ci.org/browse/INFRA-2480

### CODE REVIEW

**Review by markjacksonfishing — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -120,7 +120,7 @@ void withMavenEnv(List envVars = [], def buildType, def javaVersion, def body) {
     // Using the "tool" Workflow call automatically installs those tools on the
     // node.
     String mvntool = tool name: "mvn", type: 'hudson.tasks.Maven$MavenInstallation'
-    String jdktool = tool name: "jdk${javaVersion}", type: 'hudson.model.JDK'
+    String jdktool = tool name: "jdk8-adopt", type: 'hudson.model.JDK'
```
Comment (MarkEWaite) [inline Jenkinsfile]: We should test Adopt for both Java 8 and Java 11, shouldn't we?
```suggestion
    String jdktool = tool name: "jdk${javaVersion}-adopt", type: 'hudson.model.JDK'
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
I don't think we want to merge this until it also tests Java 11.

File: Jenkinsfile (hunk 1 above)
Comment (timja) [reply Jenkinsfile]: we already use it on 11 @MarkEWaite.

This just checks the change from oracle ~8.160 to Adopt ~2.240

**Review by res0nance — CHANGES_REQUESTED** [review]
Log seems to still say
```
[2020-02-20T20:24:32.585Z] Maven home: /usr/share/maven
[2020-02-20T20:24:32.585Z] Java version: 1.8.0_232, vendor: Oracle Corporation, runtime: /usr/local/openjdk-8/jre
[2020-02-20T20:24:32.585Z] Default locale: en, platform encoding: UTF-8
[2020-02-20T20:24:32.585Z] OS name: "linux", version: "4.15.0-1071-azure", arch: "amd64", family: "unix"
```

**Comment by timja** [comment]
That’s the aci version in the container, some of the build is run on VM and some of it is run in a docker image

**Pull request closed without merging**